index.html

```diff
@@ -1,53 +1,264 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
+  <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <link
+      href="https://fonts.googleapis.com/css?family=Poppins"
+      rel="stylesheet"
+    />
     <title>mobile-view</title>
-    
-    <link rel="stylesheet" href="style.css">
-    
-</head>
-<body>    
-    <main class="wrapper">
-        <div class="wrapper1">
-          <header>
-            <div class="logo"><a href="index.html">My Logo</a></div>
-            <nav>
-              <a href="">
-                <img id="menu1" src="./images/menu1.png" alt="menu" />
-              </a>
-            </nav>
-          </header>
-          <section class="details">
-            <h1>I'm Yusuf Glad to see you!</h1>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <div class="wrapper">
+      <div class="wrapper1">
+        <header>
+          <div class="logo"><a href="index.html">My Logo</a></div>
+          <nav>
+            <a href="">
+              <img id="menu1" src="images/menu1.png" alt="menu" />
+            </a>
```

Comment:
kindly add your images folder to your repo. This would make it easier to view your burger in the engine room after the project is cloned. I see you have selected your icon which is in your images folder. but you skipped this folder among the folders you read in the repository.

Reply:
Kindly add some light on this because I see my image folder already in the repo.

- Could it be an issue with that image in particular?